lang/python/borgbackup/Makefile

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (C) 2022 Julien Malik <julien.malik@paraiso.me>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=borgbackup
+PKG_VERSION:=1.2.2
+PKG_RELEASE:=1
+
+PYPI_NAME:=borgbackup
+PKG_HASH:=d730687443f1beb602b1d72bae36318f6f9654818fcdc50458540ec579e57260
+
+PKG_LICENSE:=BSD
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=The Borg Collective
+
+include $(TOPDIR)/feeds/packages/lang/python/pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include $(TOPDIR)/feeds/packages/lang/python/python3-package.mk
+
+define Package/borgbackup
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Deduplicated, encrypted, authenticated and compressed backups
+  URL:=https://github.com/borgbackup/borg
+  DEPENDS:= \
+      +python3-light \
+      +python3-msgpack \
+      +python3-pyfuse3 \
+      +libacl \
+      +libopenssl \
+      +liblz4 \
+      +libzstd \
+      +libxxhash
+endef
+
+define Package/borgbackup/description
```

Comment:
Two spaces for this entire define as description to be in sync with other packages.